src/Kunstmaan/AdminBundle/Resources/ui/js/_characters-left.js

```diff
@@ -0,0 +1,52 @@
+var kunstmaanbundles = kunstmaanbundles || {};
+
+kunstmaanbundles.charactersLeft = (function(window, undefined) {
+
+    var init, reInit, _updateCounter;
+
+    // Initialize
+    init = function() {
+        $('.js-max-length-input').each(function() {
+            _updateCounter(this);
+
+            $(this).on('input', _updateCounter);
+        });
+    };
+
+    reInit = function(el) {
+        if (el) {
+            _updateCounter(el);
+            $(el).on('input', _updateCounter);
+        } else {
+            $('.js-max-length-input').each(function() {
+                _updateCounter(this);
+
+                $(this).on('input', _updateCounter);
```

Comment:
Needs a check so you don't add double event listeners to items that are already initalized